sql/src/test/java/io/crate/protocols/postgres/PostgresWireProtocolTest.java

```diff
@@ -169,6 +172,76 @@ public void testBindMessageCanBeReadIfTypeForParamsIsUnknown() throws Exception
     }
 
     @Test
+    public void testDescribeMessage() throws Exception {
+        PostgresWireProtocol ctx =
+            new PostgresWireProtocol(
+                sqlOperations,
+                new AlwaysOKNullAuthentication(),
+                null);
+
+        EmbeddedChannel channel = new EmbeddedChannel(ctx.decoder, ctx.handler);
+        {
```

Comment:
I'd prefer separate test methods - makes it easier to debug in case that for example in the 2nd or 3rd block is an issue.